go.mod

```diff
@@ -5,22 +5,21 @@ go 1.19
 replace github.com/spf13/cobra => github.com/spf13/cobra v1.4.0
 
 require (
-	github.com/ONSdigital/dp-api-clients-go/v2 v2.228.0
+	github.com/ONSdigital/dp-api-clients-go/v2 v2.252.1
 	github.com/ONSdigital/dp-cache v0.1.0
 	github.com/ONSdigital/dp-component-test v0.7.0
 	github.com/ONSdigital/dp-cookies v0.3.3
-	github.com/ONSdigital/dp-healthcheck v1.5.0
-	github.com/ONSdigital/dp-net v1.5.0
-	github.com/ONSdigital/dp-net/v2 v2.8.1
-	github.com/ONSdigital/dp-renderer v1.62.0
+	github.com/ONSdigital/dp-healthcheck v1.6.1
+	github.com/ONSdigital/dp-net/v2 v2.9.1
+	github.com/ONSdigital/dp-renderer/v2 v2.0.3
```

Comment:
One of the breaking changes from v2 affects the breadcrumbs used in the search.tmpl file. 

https://github.com/ONSdigital/dp-renderer/releases/tag/v2.0.1

Reply:
Has lead to double breadcrumb (can be seen by running in Web mode and accessing /census/find-a-dataset

Reply:
Good spot. Remove breadcrumbs partial